Scripts/dea_classificationtools.py

```diff
@@ -1008,93 +1011,100 @@ def spatial_clusters(coordinates, method='Hierarchical', max_distance=None, n_gr
-
     Returns
     -------
      labels : array, shape [n_samples,]
         Index of the cluster each sample belongs to.
-
     """
     if method not in ['Hierarchical', 'KMeans', 'GMM']:
         raise ValueError(
             "method must be one of: 'Hierarchical','KMeans' or 'GMM'")
 
     if (method in ['GMM', 'KMeans']) & (n_groups is None):
         raise ValueError(
-            "The 'GMM' and 'KMeans' methods requires explicitly setting 'n_groups'")
+            "The 'GMM' and 'KMeans' methods requires explicitly setting 'n_groups'"
+        )
 
     if (method == 'Hierarchical') & (max_distance is None):
         raise ValueError(
             "The 'Hierarchical' method requires setting max_distance")
 
     if method == 'Hierarchical':
-        cluster_label=AgglomerativeClustering(n_clusters=None, linkage='complete',
-                                distance_threshold=max_distance, **kwargs).fit_predict(coordinates)
+        cluster_label = AgglomerativeClustering(
+            n_clusters=None,
+            linkage='complete',
+            distance_threshold=max_distance,
+            **kwargs).fit_predict(coordinates)
 
     if method == 'KMeans':
-        cluster_label=KMeans(n_clusters=n_groups,
+        cluster_label = KMeans(n_clusters=n_groups,
                                **kwargs).fit_predict(coordinates)
 
     if method == 'GMM':
-        cluster_label=GaussianMixture(n_components=n_groups,
+        cluster_label = GaussianMixture(n_components=n_groups,
                                         **kwargs).fit_predict(coordinates)
-
-    print("n clusters = " + str(len(np.unique(cluster_label))))
+    if verbose == True:
+        print("n clusters = " + str(len(np.unique(cluster_label))))
 
     return cluster_label
 
 
-def SKCV(coordinates, n_splits, cluster_method, kfold_method,
-         test_size, balance, n_groups=None, max_distance=None, train_size=None,
-         random_state=None, **kwargs):
+def SKCV(coordinates,
+         n_splits,
+         cluster_method,
+         kfold_method,
+         test_size,
+         balance,
+         n_groups=None,
+         max_distance=None,
+         train_size=None,
+         random_state=None,
+         **kwargs):
     """
     Generate spatial k-fold cross validation indices using coordinate data.
     This function wraps the 'SpatialShuffleSplit' and 'SpatialKFold' classes.
     These classes ingest coordinate data in the form of an
     np.array([[Eastings, northings]]) and assign samples to a spatial cluster
     using either a KMeans or Gaussain Mixture model algorithm.
-
     This cross-validator is preferred over other sklearn.model_selection methods
     for spatial data to avoid overestimating cross-validation scores.
     This can happen because of the inherent spatial autocorrelation that is usually
     associated with this type of data.
-
     Last modified: September 2020
-
     Parameters
     ----------
+    n_groups : int
```

Comment:
It'd be nice if these parameter documentations were in the same order as they are in the function